Code/D/CONTRIBUTING.md

```diff
@@ -0,0 +1,34 @@
+# How to become a contributor and submit your own code
+
+## Reporting Bugs
+
+We'd love to accept your issues and (especially) patches! If you have found a
+bug you can submit an issue describing it and one of the repo maintainers will
+get back to you. Alternately you can submit a PR and the sections below outline
+the process.
+
+## Contributing a Patch
+
+1. Fork the desired repo, develop and test your code changes.
+1. Ensure that your code has an appropriate set of unit tests which all pass.
+1. Break your changes into a series of logical self-contained commits.
+1. Submit your pull request against the appropriate branch including any
```

Comment:
Actually, as the neptune doc contains more than just guidelines about branch selection, maybe say:
"
4. For Neptune-versioned libraries, make sure you also follow the [Neptune Library Contributions Guide](https://github.com/sociomantic-tsunami/neptune/blob/master/doc/library-contributor.rst).
5. Submit your pull request against the appropriate branch.
"

Reply:
Sounds better, will change